### ISSUE TITLE
Failure to parse the data transmitted with get method and Internal Server Error

### ISSUE THREAD

**Issue by hakanuzum** [denisov-vlad/simple-snowplow#22]
Hi, Vlad, 
I get Internal server error when sending data with get method. For example, snowplow integration for AMP pages sends data with get method. 

When the get method sends data "collectorHost/i?..." loads data in the form. 

![image](https://github.com/denisov-vlad/simple-snowplow/assets/16476982/5fd841c9-9c4a-4f00-9785-a82e64e8934f)

In the image above, the data received with the get method is received by /i. And while the incoming data is being parse, the same payload model is used as in post submissions. 

![image](https://github.com/denisov-vlad/simple-snowplow/assets/16476982/6725d4bc-6dad-43e0-a46c-c5fe13aab19c)

So basically, in order to get the data from the get method, I think I should write a different model instead of using "PayloadElementBaseModel" and parse the data coming from the get method with it. 

![image](https://github.com/denisov-vlad/simple-snowplow/assets/16476982/6ba10dc8-8200-491a-93d4-f142e3160fb6)

I encountered such a problem with Snowplow amp integration.

**Comment by denisov-vlad** [comment]
Hi Hakan.

Thanks for the report, I'll try to fix it till the end of the week.

**Comment by denisov-vlad** [comment]
@hakanuzum Could you please check with the updated master branch? I think I fixed it in the bd82e71 commit, but can't verify it on the real AMP page.
